scipy/special/Faddeeva.cc

```diff
@@ -125,7 +125,7 @@ extern "C" {
 #include <cfloat>
 #include <cmath>
 
-using namespace std;
+#define complex std::complex
```

Comment:
Maybe also depends on whether this is taken over upstream. If not, I'd rather keep the diff as small as possible. Let's first see that @tyhone confirms that this really works, then decide which is preferable?